### PR TITLE
Dev 1.3

### PR DESCRIPTION
**Pull request by raydittmeier**
This is for ST 1.3, which includes 2 fixes, cleanup of negative emissivity values from bad values in the ASTER GED database (likely due to undetected clouds in the ASTER data), and ensuring that required NARR points are used along the top and bottom of the scene by detecting the scene edges there more thoroughly.  This is not going into ESPA soon, but it will be used by L2PGS soon so I think it should be reviewed.  The pull request is showing the ST 1.2 changes for the review, I think because those haven't been released to ESPA yet, but they have already been reviewed and there is no need to review them again.  The review is really on the README, st_utilities.py, const.h, estimate_landsat_emissivity.py, and st_determine_grid_points.py.

### CODE REVIEW

**Comment by gschmidt-usgs** [comment]
Line 258 in emissivity_utilities.py refers to TOA BT.  This would be better if it were referred to simply at BT.

File: provisional-st/scripts/estimate_landsat_emissivity.py [hunk 1]
```diff
@@ -820,6 +820,7 @@ def generate_emissivity_data(xml_filename, server_name, server_path,
     ls_emis_final[aster_ndvi_gap_locations] = no_data_value
     ls_emis_final[ndvi_no_data_locations] = no_data_value
     ls_emis_final[ndsi_no_data_locations] = no_data_value
+    ls_emis_final[ls_emis_final < 0.0] = no_data_value
```
Comment (gschmidt-usgs) [inline provisional-st/scripts/estimate_landsat_emissivity.py]: The comment above this code block could use some help in specifying what is happening in this code block.  It looks like many pixels are getting set to fill if they meet certain criteria (such as the newly added emissivity being less than zero).

Comment (gschmidt-usgs) [reply provisional-st/scripts/estimate_landsat_emissivity.py]: No further comment.

Comment (raydittmeier) [reply provisional-st/scripts/estimate_landsat_emissivity.py]: I updated the comment with more details, and took out "TOA" from "TOA BT".

**Pull request merged**